src/lib/os-release.ts

```diff
@@ -0,0 +1,47 @@
+import * as Promise from 'bluebird';
```

Comment:
I think we should start importing Bluebird as Bluebird so that we don't override Promise anymore. This will allow an easier migration to async/await.

Reply:
Yep, this will definitely save some headaches in the future